src/main/java/domain/Winner.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winner {
+    public static String makeWinnerToString(List<Car> cars) {
+        StringBuilder winner = new StringBuilder(cars.get(0).getName());
+
+        if(cars.size() > 1) {
+            for(int i = 1;i<cars.size();i++) {
+                winner.append(", ").append(cars.get(i).getName());
+            }
+        }
```

Comment:
```suggestion
        if (cars.size() > 1) {
            for (int i = 1; i < cars.size(); i++) {
                winner.append(", ").append(cars.get(i).getName());
            }
        }
```

for문과 if문 사이의 공백도 컨벤션입니다 🥲
IDE의 reformat 기능을 활용해보세요 🙂